moncli/entities/item.py

```diff
@@ -336,17 +345,29 @@ def remove_files(self, file_column: en.cv.FileValue, *args):
                 url_thumbnail : `str`
                     Url to view the asset in thumbnail mode. Only available for images.
         """
+        value = json.dumps({ "clear_all": True} )
```

Comment:
This causes an Internal Server Error from the Monday.com API.  No need to serialize this here.  That happens inside the api handler function.  I would suggest passing in the dict as the value directly in the function call below.